builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -82,6 +82,11 @@ node() {
 
     if (configObj.service_id) {
       config = serviceMetadataLoader.loadServiceMetadata(configObj.service_id)
+      if (config['domain'] && config['domain'] == "jazz") {
```

Comment:
SAme ass above comment. Cut this lines of code from 85 to 89 from here and paste it at line number 98